src/v0/destinations/google_adwords_offline_conversions/networkHandler.js

```diff
@@ -37,6 +41,43 @@ const getAuthErrCategory = (status) => {
   }
 };
 
+const createJob = async (endpoint, headers, payload) => {
+  const endPoint = `${endpoint}:create`
+  let createJobResponse = await httpPOST(endPoint, payload, { headers });
+  createJobResponse = processAxiosResponse(createJobResponse);
+  const { response, status } = createJobResponse;
+  if (!isHttpStatusSuccess(status)) {
+    throw new AbortedError(
+      `[Google Ads Offline Conversions]:: ${response?.error?.message} during google_ads_offline_store_conversions Job Creation`,
+      status,
+      response,
+      getAuthErrCategory(status),
+    );
+  }
+  return response.resourceName.split('/')[3];
+};
+
+const addUserToJob = async (endpoint, headers, jobId, payload) => {
```

Comment:
```suggestion
const addUserToJob = async (endpoint, headers, jobId, payload) => {
```
Can we update the function name with something like add conversionToJob?

Reply:
Done